Models/Connect/ProductToRemoteCategoryRepository.php

```diff
@@ -158,8 +158,8 @@ public function deleteByConnectCategoryId($categoryId, $articleId)
         $builder->delete('Shopware\CustomModels\Connect\ProductToRemoteCategory', 'ptrc');
         $builder->where('ptrc.connectCategoryId = :ccid');
         $builder->andWhere('ptrc.articleId = :articleId');
-        $builder->setParameter(':ccid', $categoryId);
-        $builder->setParameter(':articleId', $articleId);
+        $builder->setParameter(':ccid', (int) $categoryId);
+        $builder->setParameter(':articleId', (int) $articleId);
```

Comment:
\PDO::PARAM_INT here and the line above too
Sorry i missed that in the first review